server/sonar-db-dao/src/test/java/org/sonar/db/qualitygate/QualityGateDaoTest.java

```diff
@@ -101,6 +101,31 @@ public void testSelectById() {
     assertThat(underTest.selectById(dbSession, 42L)).isNull();
   }
 
+  @Test
+  public void select_by_organization_and_uuid() {
+    OrganizationDto organization = db.organizations().insert();
+    QualityGateDto qualityGate = db.qualityGates().insertQualityGate(organization);
+    OrganizationDto otherOrganization = db.organizations().insert();
+    QualityGateDto otherQualityGate = db.qualityGates().insertQualityGate(otherOrganization);
+
+    assertThat(underTest.selectByOrganizationAndUuid(dbSession, organization, qualityGate.getUuid()).getUuid()).isEqualTo(qualityGate.getUuid());
+    assertThat(underTest.selectByOrganizationAndUuid(dbSession, otherOrganization, qualityGate.getUuid())).isNull();
+    assertThat(underTest.selectByOrganizationAndUuid(dbSession, organization, otherQualityGate.getUuid())).isNull();
+  }
+
+  @Test
+  public void select_by_organization_and_name() {
+    OrganizationDto organization = db.organizations().insert();
+    QualityGateDto sonarWay = db.qualityGates().insertQualityGate(organization, g -> g.setName("Sonar way"));
```

Comment:
I think we should use another name for the test (because we are expecting "Sonar way" to be builtIn and unique).